modules/040-node-manager/openapi/values.yaml

```diff
@@ -128,13 +130,13 @@ properties:
 
       static:
         type: object
-        default: {}
+        default: { }
         description: |
           Settings for Static nodes.
         properties:
           internalNetworkCIDRs:
             type: array
-            default: []
+            default: [ ]
```

Comment:
If it is some standard — we should upgrade all the openapis in all the modules at a time.